catroid/src/org/catrobat/catroid/content/Sprite.java

```diff
@@ -274,20 +282,21 @@ public void setActionFactory(ActionFactory actionFactory) {
 
 	@Override
 	public Sprite clone() {
```

Comment:
Not a suggestion, just a note:

I'd be glad if we could start merging our features as soon as possible. I suppose that my Cloning feature (PR #1823) has to be  adapted to the grouping feature as well (and a lot of the PRs might have conflicts with other PRs). I am affraid that 10 hours won't suffice for that. :warning: